iot-hub/Samples/device/PnpDeviceSamples/TemperatureController/Program.cs

```diff
@@ -73,7 +73,12 @@ public static async Task Main(string[] args)
             s_logger.LogDebug($"Set up the device client.");
             using DeviceClient deviceClient = await SetupDeviceClientAsync(parameters, cts.Token);
             var sample = new TemperatureControllerSample(deviceClient, s_logger);
-            await sample.PerformOperationsAsync(cts.Token);
+
+            try
+            {
+                await sample.PerformOperationsAsync(cts.Token);
+            }
+            catch (OperationCanceledException) { }
```

Comment:
nit: you could add a note in there that explains why the OperationCanceledException is expected.